testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/env/environment/EnvMultinodeAzure.java

```diff
@@ -105,6 +120,21 @@ private Path getCoreSiteOverrideXml()
         }
     }
 
+    private Path getTemptoConfiguration(String schema)
+    {
+        try {
+            String contents = Files.readString(configDir.getPath("tempto-configuration-abfs-template.yaml"))
+                    .replace("%ABFS_SCHEMA%", schema);
+            File temptoConfiguration = Files.createTempFile("tempto-configuration", ".yaml", PosixFilePermissions.asFileAttribute(fromString("rwxrwxrwx"))).toFile();
```

Comment:
I agree, but could we do that in a follow-up PR? We'll continue working on switching any remaining GCS tests to native FS, and maybe it would be easier to port this test and have it run for different storage backends.

Reply:
Yeah, I'm not suggesting any changes here. Just commenting that this shows how horrible it is to use this framework. I'd be quite happy to see a follow up here that reduces usage of Tempto.

Reply:
In "normal" tests we also have cases where we write (XML) configuration files
https://github.com/trinodb/trino/blob/ddf931f65c93f53ce46dfb70d3efa2cfc66b2b7a/plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestDeltaLakeAdlsStorage.java#L74

i hope we can remove these too, it's indeed horrible.